app/src/main/scala/org/alephium/explorer/api/model/UnsignedTransaction.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Alephium
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package org.alephium.explorer.api.model
+
+import scala.collection.immutable.ArraySeq
+
+import sttp.tapir.Schema
+
+import org.alephium.api.TapirSchemas._
+import org.alephium.explorer.api.Json._
+import org.alephium.json.Json._
+import org.alephium.protocol.model.TransactionId
+import org.alephium.util.U256
+
+final case class UnsignedTransaction(
+    hash: TransactionId,
+    version: Byte,
+    networkId: Byte,
+    scriptOpt: Option[String],
+    gasAmount: Int,
+    gasPrice: U256,
+    inputs: ArraySeq[Input],
+    outputs: ArraySeq[Output]
+) {
+  def chainFrom: Option[Int] =
```

Comment:
Shall we return `Int` instead of `Option[Int]`? Since we should always have `chainFrom`.

Reply:
well `chainFrom` is derived from the inputs [as in the node](https://github.com/alephium/alephium/blob/master/protocol/src/main/scala/org/alephium/protocol/model/UnsignedTransaction.scala#L58-L60), but as I don't control the data here, I thought it was safer to use `Option` in case inputs are empty.

But actually I realized that `chainFrom` isn't used, can't remember why it was introduced. I will simply remove it for now.